openid-sse-framework-1_0.txt

```diff
@@ -557,39 +552,44 @@ Tulshibagwale, et al.   Expires December 10, 2021               [Page 9]
 
 
 
-Tulshibagwale, et al.   Expires December 10, 2021              [Page 10]
+
+
+
+
+
+Tulshibagwale, et al.        Standards Track                   [Page 10]
 
                           openid-sse-framework                 June 2021
 
 
    Transmitters supporting Discovery MUST make a JSON document available
-   at the path formed by inserting the string "/.well-known/sse-
-   configuration" into the Issuer between the host component and the
-   path component, if any.  The syntax and semantics of ".well-known"
-   are defined in [RFC5785].  "sse-configuration" MUST point to a JSON
+   at the path formed by inserting the string /.well-known/sse-
+   configuration into the Issuer between the host component and the path
+   component, if any.  The syntax and semantics of .well-known are
+   defined in [RFC5785].  sse-configuration MUST point to a JSON
    document compliant with this specification and MUST be returned using
-   the "application/json" content type.
+   the application/json content type.
 
 6.2.1.  Transmitter Configuration Request
 
    A Transmitter Configuration Document MUST be queried using an HTTP
-   "GET" request at the previously specified path.
+   GET request at the previously specified path.
 
    The Receiver would make the following request to the Issuer
-   "https://tr.example.com" to obtain its Configuration information,
-   since the Issuer contains no path component:
+   https://tr.example.com to obtain its Configuration information, since
+   the Issuer contains no path component:
 
    GET /.well-known/sse-configuration HTTP/1.1
    Host: tr.example.com
 
     Figure 9: Example: Transmitter Configuration Request (without path)
 
-   If the Issuer value contains a path component, any terminating "/"
```

Comment:
I've noticed that the quotes are gone from many places in the spec. Is that intentional or an artifact of the tools you are using?